src/metabase/pulse/render/body.clj

```diff
@@ -601,66 +567,98 @@
         settings     (->ts-viz (first x-cols) (first y-cols) labels viz-settings)]
     (attach-image-bundle (image-bundle/make-image-bundle render-type (js-svg/combo-chart series settings)))))
 
-(s/defmethod render :multiple
-  [_ render-type _timezone-id card dashcard data]
-  ((if (= :scalar (:display card))
-     render-multiple-scalars
-     render-multiple-lab-chart)
-   render-type card dashcard data))
-
 (defn- series-setting [viz-settings outer-key inner-key]
   (get-in viz-settings [:series_settings (keyword outer-key) inner-key]))
 
 (defn- single-x-axis-combo-series
   "This munges rows and columns into series in the format that we want for combo staticviz for literal combo displaytype,
   for a single x-axis with multiple y-axis."
-  [chart-type joined-rows _x-cols y-cols viz-settings]
+  [chart-type joined-rows _x-cols y-cols viz-settings card-name]
   (for [[idx y-col] (map-indexed vector y-cols)]
     (let [y-col-key     (keyword (:name y-col))
-          card-name     (or (series-setting viz-settings y-col-key :name)
-                            (series-setting viz-settings y-col-key :title)
-                            (:display_name y-col))
-          card-color    (or (series-setting viz-settings y-col-key :color)
-                            (nth colors idx))
+          metric-name          (or (series-setting viz-settings y-col-key :name)
+                                   (series-setting viz-settings y-col-key :title))
           card-type     (or (series-setting viz-settings y-col-key :display)
                             chart-type
                             (nth default-combo-chart-types idx))
           selected-rows (mapv #(vector (ffirst %) (nth (second %) idx)) joined-rows)
           y-axis-pos    (or (series-setting viz-settings y-col-key :axis)
                             (nth (default-y-pos viz-settings) idx))]
-      {:name          card-name
-       :color         card-color
+      {:name          metric-name
```

Comment:
In summary

- `single-x-axis-combo-series` will be called when a question has only 1 dimension.
- And `double-x-axis-combo-series` will be called when a question has 2 dimensions.

With question with 1 dimension. We need:
- name: (custom metric name)
- cardName: (or a question name)

And with question with 2 dimensions. We need:
- name: (custom column name)
- cardName: (or a question name)
- breakoutValue: e.g. `2016-01-01T00:00:00Z` for FE to calculate the series key as `2016`